db/migrate/20150115050935_add_addresses_ref_to_enterprise_groups.rb

```diff
@@ -0,0 +1,6 @@
+class AddAddressesRefToEnterpriseGroups < ActiveRecord::Migration
+  def change
+    add_column :enterprise_groups, :address_id, :integer
+    add_foreign_key :enterprise_groups, :spree_addresses, name: "enterprise_groups_address_id_fk", column: "address_id"
```

Comment:
Can we leave out the name attribute here? Or is it required in this case?

Also, it'd be good to add an index on this key:

```ruby
add_index :enterprise_groups, :address_id
```